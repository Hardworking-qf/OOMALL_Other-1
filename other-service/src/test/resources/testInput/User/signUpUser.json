{
  "1": {
    "userName": "testuser1",
    "password": "123456",
    "realName": "test",
    "mobile": "12300010003",
    "email": "test@test1.com",
    "birthday": "2020-01-01",
    "gender": 0
  },
  "2": {
    "userName": "testuser",
    "password": "123456",
    "realName": "test",
    "mobile": "12300010003",
    "email": "test@test1.com",
    "birthday": "2020-01-01",
    "gender": 0
  },
  "3": {
    "userName": "testuser1",
    "password": "123456",
    "realName": "test",
    "mobile": "12300010001",
    "email": "test@test1.com",
    "birthday": "2020-01-01",
    "gender": 0
  },
  "4": {
    "userName": "testuser1",
    "password": "123456",
    "realName": "test",
    "mobile": "12300010003",
    "email": "test@test.com",
    "birthday": "2020-01-01",
    "gender": 0
  },
  "5": {
    "userName": "",
    "password": "123456",
    "realName": "test",
    "mobile": "12300010002",
    "email": "test@test.com",
    "birthday": "2020-01-01",
    "gender": 0
  },
  "6": {
    "userName": "testuser1",
    "password": "",
    "realName": "test",
    "mobile": "12300010002",
    "email": "test@test.com",
    "birthday": "2020-01-01",
    "gender": 0
  },
  "7": {
    "userName": "testuser1",
    "password": "123456",
    "realName": "",
    "mobile": "12300010002",
    "email": "test@test.com",
    "birthday": "2020-01-01",
    "gender": 0
  },
  "8": {
    "userName": "testuser1",
    "password": "123456",
    "realName": "test",
    "mobile": "",
    "email": "test@test.com",
    "birthday": "2020-01-01",
    "gender": 0
  },
  "9": {
    "userName": "testuser1",
    "password": "123456",
    "realName": "test",
    "mobile": "12300010002asdasd",
    "email": "test@test.com",
    "birthday": "2020-01-01",
    "gender": 0
  },
  "10": {
    "userName": "testuser1",
    "password": "123456",
    "realName": "test",
    "mobile": "12300010002",
    "email": "",
    "birthday": "2020-01-01",
    "gender": 0
  },
  "11": {
    "userName": "testuser1",
    "password": "123456",
    "realName": "test",
    "mobile": "12300010002",
    "email": "tests546^&(*)test.com",
    "birthday": "2020-01-01",
    "gender": 0
  },
  "12": {
    "userName": "testuser1",
    "password": "123456",
    "realName": "test",
    "mobile": "12300010002",
    "email": "test@test1.com",
    "birthday": "",
    "gender": 0
  },
  "13": {
    "userName": "testuser1",
    "password": "123456",
    "realName": "test",
    "mobile": "12300010002",
    "email": "test@test1.com",
    "birthday": "2020-19-01",
    "gender": 0
  },
  "14": {
    "userName": "testuser1",
    "password": "123456",
    "realName": "test",
    "mobile": "12300010002",
    "email": "test@test1.com",
    "birthday": "2040-01-01",
    "gender": 0
  },
  "15": {
    "userName": "testuser1",
    "password": "123456",
    "realName": "test",
    "mobile": "12300010002",
    "email": "test@test1.com",
    "birthday": "2020-01-01",
    "gender": ""
  },
  "16": {
    "userName": "testuser1",
    "password": "123456",
    "realName": "test",
    "mobile": "12300010002",
    "email": "test@test1.com",
    "birthday": "2020-01-01",
    "gender": 20
  }
}