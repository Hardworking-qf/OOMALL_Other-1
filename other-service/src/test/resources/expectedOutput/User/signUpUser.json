{
  "1": {
    "errno": 0,
    "data": {
      "id": 17334,
      "userName": "testuser1",
      "realName": "test",
      "mobile": "12300010003",
      "email": "test@test1.com",
      "gender": 0,
      "birthday": "2020-01-01",
      "gmtCreate": "2020-11-30T15:45:50.030396",
      "gmtModified": null
    },
    "errmsg": "成功"
  },
  "2": {
    "errno": 731,
    "errmsg": "用户名已被注册"
  },
  "3": {
    "errno": 733,
    "errmsg": "手机号已被注册"
  },
  "4": {
    "errno": 732,
    "errmsg": "邮箱已被注册"
  }
}